lib/src/macos_app.dart

```diff
@@ -65,100 +95,196 @@ class MacosApp extends StatefulWidget {
     this.checkerboardRasterCacheImages = false,
     this.checkerboardOffscreenLayers = false,
     this.showSemanticsDebugger = false,
-    this.debugShowWidgetInspector = false,
     this.debugShowCheckedModeBanner = true,
-    this.inspectorSelectButtonBuilder,
     this.shortcuts,
     this.actions,
-  })  : assert(routeInformationParser != null && routerDelegate != null,
-            'The routeInformationParser and routerDelegate cannot be null.'),
-        assert(supportedLocales.isNotEmpty),
+    this.restorationScopeId,
+    this.themeMode,
+    this.style,
+    this.darkStyle,
+  })  : assert(supportedLocales.isNotEmpty),
         navigatorObservers = null,
-        backButtonDispatcher =
-            backButtonDispatcher ?? RootBackButtonDispatcher(),
         navigatorKey = null,
         onGenerateRoute = null,
-        pageRouteBuilder = null,
         home = null,
         onGenerateInitialRoutes = null,
         onUnknownRoute = null,
         routes = null,
         initialRoute = null,
         super(key: key);
 
-  final Map<Type, Action<Intent>>? actions;
-
-  final BackButtonDispatcher? backButtonDispatcher;
-
-  final TransitionBuilder? builder;
-
-  final bool checkerboardOffscreenLayers;
+  /// {@macro flutter.widgets.widgetsApp.navigatorKey}
+  final GlobalKey<NavigatorState>? navigatorKey;
```

Comment:
Sort class members alphabetically, per code_metrics analysis

Reply:
Why? From my point of view this is useless. Just extra unecessary work

Reply:
Alright, no worries